src/dnx.host/Bootstrapper.cs

```diff
@@ -76,6 +76,7 @@ public Task<int> RunAsync(List<string> args, IRuntimeEnvironment env)
                 serviceProvider.Add(typeof(IRuntimeEnvironment), env);
 
                 CallContextServiceLocator.Locator.ServiceProvider = serviceProvider;
+                var mono = RuntimeEnvironmentHelper.IsMono;
```

Comment:
I should add a comment. This is my proposed temporary fix for mono null-ref. If we grab the value here it wont fail later.